django/template/loaders/cached.py

```diff
@@ -82,7 +82,7 @@ def cache_key(self, template_name, template_dirs, skip=None):
         return ("%s-%s-%s" % (template_name, skip_prefix, dirs_prefix)).strip('-')
 
     def generate_hash(self, values):
-        return hashlib.sha1(force_bytes('|'.join(values))).hexdigest()
```

Comment:
And what's the benefit of the change? Just because sha1 is susceptible to collision doesn't necessarily mean there is some problem with using it in this context.